src/server/migrations/20201216124500_drop_2_fk_columns_from_feedback.js

```diff
@@ -0,0 +1,11 @@
+exports.up = function (knex) {
+  return knex.schema.table('feedback', function (table) {
+    table.dropForeign('answered_by');
+    table.dropColumn('answered_by');
+    table.dropForeign('question_id');
+    table.dropColumn('question_id');
+  });
+};
+exports.down = function (knex) {
+  return knex.schema.table('feedback');
```

Comment:
You need to recreate columns `answered_by` and `question_id`

Reply:
The same as above